src/pages/en/guides/cms/storyblok.mdx

```diff
@@ -368,29 +366,54 @@ export default defineConfig({
 
 ### Generating pages
 
+To create a route for a specific `page`, you can fetch its directly from the Storyblok API and pass it to the `Page` component. Create an `index.astro` file in `src/pages` to render the `home` page:
+
+```astro title="src/pages/index.astro" {7,8,15} 
+---
+import { useStoryblokApi } from '@storyblok/astro'
+import BaseLayout from '../layouts/BaseLayout.astro'
+import Page from '../storyblok/Page.astro';
+
+const storyblokApi = useStoryblokApi();
+const { data } = await storyblokApi.get('cdn/stories/home');
+const content = data.story.content;
+---
+<html lang="en">
+  <head>
+    <title>Storyblok & Astro</title>
+  </head>
+  <body>
+    <Page blok={content} />
+  </body>
+</html>
+```
+
+To dynamically render all of the blog posts, create a dynamic route. This approach varies depending on whether you're using **static site generation** (the default) or **server-side rendering**.
```

Comment:
```suggestion
To generate pages for all of your blog posts, create a `.astro` page that will create dynamic routes. This approach varies depending on whether you're using **static site generation** (the default) or **server-side rendering**.
```
Just thinking that the reader doesn't care that "blog posts are dynamically generated" so much as they care that "a page is made for each blog post"? Plus, this avoids using the word "dynamic" twice. :smile: (And, I'm not sure I'd say *a* dynamic route is created, but a page that creates a bunch of routes.) Tweak this as you like!